sig-documentation/README.md

```diff
@@ -0,0 +1,45 @@
+# Spinnaker Documentation Special Interest Group
+
+## Roles & Responsibilities of the SIG Leads
+
+### SIG Lead(s):
+
+**[Brian Le](https://github.com/brian-armory)**  
+Technical Writer at Armory
+
+**[Ben Klein](https://github.com/fifthposition)**  
+Technical Writer at Pivotal
+
+**[David Dorbin](https://github.com/dorbin)**  
+Technical Writer at Google
+
+### SIG Lead Responsibility
+
+1. Coordinate communication.
+2. Set meeting dates and objectives for the coming year.
+3. Curate agenda for each meeting.
+4. Raise awareness and invite appropriate participants.
+5. Encourage and foster an inclusive environment.
+6. Facilitate the identification and development of resources (people or funding) in support of documentation for Spinnaker.
+
+
+### Meeting Cadence
+ - 1 hour every month on Google Hangouts
+ - TBD of every month at TBD PM PST
+
+# Responsibilities
+
+## Description
+
+Documentation SIG is responsible for the direction and organization of the Spinnaker documentation on https://www.spinnaker.io/. The SIG also acts as a place where users and developers can find collaborators and get help creating or modifying documentation.
+
+SIG leads and members will develop, maintain, and prioritize a backlog of documentation projects and requests. These projects and requests are macro-level issues, such as organization. They are independent of releases and features.  This prioritization will be published as the Documentation SIG Roadmap.
+
+## Scope of Work and Goals
+
+* Encourage Spinnaker adoption by creating helpful documentation.
+* Establish a common framework for evaluating and improving documentation.
+* Develop, encourage, and maintain a clearly defined structure and organization for the Spinnaker documentation.
+* Assist developers and SIGs with creating documentation for new features in Spinnaker.
+* Propose and drive initiatives for code-driven doc generation.
+* Facilitate community contributions to the Spinnaker documentation by creating templates and style guides.
```

Comment:
Would add...
* Maintain and improve the platform for building and publishing spinnaker.io.